src/model/net.py

```diff
@@ -1,4 +1,118 @@
-"""
-    This script was made by Nick at 19/07/20.
-    To implement code of your network using operation from ops.py.
-"""
+import math
+
+import numpy as np
+import torch
+import torch.nn as nn
+
+from src.model.ops import *
+
+
+def conv_out_size(size, stride: int = 2) -> int:
+    return int(math.ceil(float(size) / float(stride)))
```

Comment:
이 함수는 `Generator`에서만 쓰이는 것 같네요!
여기에 있으면 `Generator`와 `Discriminator`모두에 있어야 할 것 같다는 인상을 주는데요,
혹시 `Generator` 클래스 안에 private 함수로 넣는 것은 어떨까요?

```suggestion
class Generator(nn.Module):
    ...
    def _conv_out_size(self, size, stride: int = 2) -> int:
        return int(math.ceil(float(size) / float(stride)))
```

Reply:
요 부분은 크게 필요하지 않을 것 같아 제거했어요!! 감사합니다